### PR TITLE
fix: modify ma-alert

### PR DESCRIPTION
**Pull request by sergigraciaj**
- Reduce icon size to fit Watt alert design
- Add margin-top to align with the first line of the text (from title, text or slot). I haven't used align-items here as the problem is a combination of icon's height and ma-text top-crop.

### CODE REVIEW

File: src/components/MaAlert/MaAlert.vue [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <template>
   <div :class="['alert', `alert--${tone}`]">
-    <ma-icon class="icon" :icon="icon" :tone="tone" />
-    <ma-stack space="xsmall">
+    <ma-icon class="icon" size="small" :icon="icon" :tone="tone" />
```
[the remark this answers is not in the document]
Comment (sergigraciaj) [reply src/components/MaAlert/MaAlert.vue:3]: Should we rely on chromatic for this kind of changes? I mean, ma-icon size prop is already tested in it's own component.

[the remark this answers is not in the document]
Comment (alex-castelo) [reply src/components/MaAlert/MaAlert.vue:3]: I'm with Sergi , what test can do here? Maybe test if we pass a random icon breaks alert icon?

[the remark this answers is not in the document]
Comment (edufarre) [reply src/components/MaAlert/MaAlert.vue:3]: I am not talking about testing the name passed to the icon as a prop, but the title of the PR is `fix` which means something was not right, and apparently this is the icon size. So I would add a test asserting that the icon size is `small`.

And yes yes yes, chromatic, but do you understand what is actually happening in the chromatic snapshot?

Screen Shot 2022-02-09 at 09 14 39

**Review by edufarre — APPROVED** [review]
🚀

**Pull request merged**